docs/tutorials/vis_cpu_tutorial.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>I think you gain very little in this tutorial by making a <code>UVData</code> object  -- you don't actually really use that object, just its inputs. I think the tutorial will be much cleaner if you instead just construct your frequencies/LSTs directly as arrays and use those as inputs, rather than going through <code>UVData</code>.  You could also add a little bit more explanatory text throughout. At the end, it may be useful to say that <code>hera_sim</code>  has a wrapper that makes many of these setup operations easier, and allows you to run <code>vis_cpu</code>  based on pyuvsim-compatible configuration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/vis_cpu/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I have modified the notebook as suggested.